categories/99-problems/P40-rhebus.pl

```diff
@@ -35,6 +35,6 @@ (Even $n where * > 2 )
     return; # fail
 }
 
-say goldbach $_ for 28, 36, 52, 110, 62710560;
+say ~ goldbach $_ for 28, 36, 52, 110;
```

Comment:
Hrm, that's odd, there should have been a corresponding expected output.  That should probably be added to the test if it wasn't there before.  I'd like to maintain the original behaviour as much as possible.